docs/tools/vdb_table/data/usearch.json

```diff
@@ -53,29 +53,29 @@
     "comment": "using the multi param in index creation"
   },
   "sparse_vectors": {
-    "support": "",
+    "support": "partial",
     "source_url": "",
     "comment": ""
   },
   "bm25": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "full_text": {
-    "support": "",
+    "support": "partial",
     "source_url": "",
-    "comment": ""
+    "comment": "with custom user-defined metrics on similar-length string until v3"
   },
   "embeddings_text": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
```

Comment:
this is only set if the embeddings can be run within the database

Reply:
I'm not sure it's a fair comparison. If the question is - can we add a one-liner in our Python bindings to invoke some predefined web-API or run a some third-party model locally, of course it can be added. That, however, would be a bad design, cramming together things, that are independent by design. Bad for both users and us.

A more meaningful question might be, did the team behind a search engine also spend years on representation learning R&D, and do they also provide models suited for retrieval.

Up to you to decide 🤗

Reply:
I understand your point-of-view. However, the meaning of this column in this table is about being able to run the embeddings within the search engine system. You can mark it as partial, since you do maintain the UForm library which does this in conjunction with USearch.